src/main/java/uk/ac/ebi/intact/app/internal/tasks/query/QueryParams.java

```diff
@@ -0,0 +1,34 @@
+package uk.ac.ebi.intact.app.internal.tasks.query;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.ac.ebi.intact.app.internal.model.core.view.NetworkView;
+
+import java.util.Set;
+
+@Data
+@Builder
+public class QueryParams {
```

Comment:
Because the `@Tunable` parameters are a pain to work with, I've created this class containing the filters and other parameters parsed and typed properly, so we can use this class on the code for filters and other stuff.